cmd/podman/stop.go

```diff
@@ -54,44 +54,68 @@ func stopCmd(c *cliconfig.StopValues) error {
 		defer span.Finish()
 	}
 
+	args := c.InputArgs
 	if err := checkAllAndLatest(&c.PodmanCommand); err != nil {
 		return err
 	}
 
 	rootless.SetSkipStorageSetup(true)
 	runtime, err := libpodruntime.GetRuntime(&c.PodmanCommand)
```

Comment:
this should be calling adapter.GetRuntime ... with the code as is, you never actually go over the varlink connection.